src/components/Sidebaritem/__test__/sidebaritem.spec.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
```

Comment:
let's use SidebarItem for the folder name of the components, just to be consistent

Reply:
also change to sidebarItem.spec.js the test file